ocd-1001.sql

```diff
@@ -4,18 +4,30 @@ TRUNCATE openchpl.surveillance CASCADE;
 -- Build temp staging tables for each destination table
 DROP TABLE IF EXISTS surveillance;
```

Comment:
Do these tables need to be in a schema? Even if they're temp tables, will they have issues on the deployment machines without a schema?